src/component/feature/strategy/strategies-list-component.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import { cloneDeep } from 'lodash';
+import arrayMove from 'array-move';
+import { Button, Icon } from 'react-mdl';
+
+import ConfigureStrategy from './strategy-configure-container';
+import AddStrategy from './strategies-add';
+import { HeaderTitle } from '../../common';
+import { updateIndexInArray } from '../../common/util';
+import styles from './strategy.module.scss';
+
+const randomKeys = length => Array.from({ length }, () => Math.random());
```

Comment:
not needed anymore as we are not allowing re-order while updating activation strategies. I will clean this up.

Reply:
fixed

Reply:
:+1: